descope/sdk/auth.go

```diff
@@ -313,6 +313,12 @@ type Authentication interface {
 	// the specified roles for a specific tenant.
 	ValidateTenantRoles(token *descope.Token, tenant string, roles []string) bool
 
+	// SelectTenantWithRequest - Use to add to the jwt a dedicated claim to mark on which tenant this user authenticated now
```

Comment:
comment is great, here is an improvement suggestion though

```suggestion
	// SelectTenantWithRequest - Adds a dedicated claim to the JWTs to indicate the tenant on which the user is currently authenticated
```

Reply:
done